sections/method/PageRank.tex

```diff
@@ -27,4 +27,7 @@ \subsection{PageRank}\label{sec:pagerank}
 where $p$ and $q$ are distributions and $m$ is the pointwise mean between $p$ and $q$. 
 $D$ is the Kullback-Leibler divergence function.
 
-The goal of using \gls{pr} is to get the most important documents within a specific topic and use these to yield important information during search.  
+The goal of using \gls{pr} is to get the most important documents within a specific topic and use these to yield important information during search.
+  
+\gls{pr} does not take the query into account, which is the reason it is only used in combination with other \gls{ir} methods. 
+The \gls{pr} is to a certain degree connected with \gls{lda} since it uses the topic distributions created by the \gls{lda} to rank documents. 
```

Comment:
```suggestion
The \gls{pr} is to a certain degree connected with \gls{lda} since it uses the topic distributions, created by the \gls{lda}, to rank documents. 
```